staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation.go

```diff
@@ -144,20 +144,29 @@ func validator(s *schema.Structural, isResourceRoot bool, declType *cel.DeclType
 	return nil
 }
 
+type CELValidationOptions struct {
+	ratchetingOptions
+}
+
+type CELValidationOption func(*CELValidationOptions)
```

Comment:
nit: Drop CEL prefix to avoid a stutter?

Reply:
renamed to `Option`